docs/run-tests-locally.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[run-tests-locally]]
+== Run Tests Locally
+
+To run tests locally you can make use of the docker images also used when running the whole test suite with Jenkins. 
+Running the full test suite first does some linting and then runs the actual tests with different versions of Python and different web frameworks. For a full overview of the test matrix and supported versions have a look at 
+https://github.com/elastic/apm-agent-python/blob/master/Jenkinsfile[Jenkins Configuration].
```

Comment:
This is linking to the github repo, not sure if that is what we want, but since the `Jenkinsfile` is not included in our docs I did not see any other option here, maybe @ruflin knows more?

Reply:
We could include it but we would have to add it as `sources` in the `conf.yaml` file: https://github.com/elastic/docs/blob/master/conf.yaml#L541 TBH I think for now it is completely fine to link to this file in the Github repo.